modules/atomic-widgets/module.php

```diff
@@ -155,4 +156,17 @@ private function enqueue_scripts() {
 			true
 		);
 	}
+
+	private function enqueue_styles() {
+		wp_register_style(
+			'div-block',
+			$this->get_css_assets_url( 'div-block', 'assets/css/' ),
```

Comment:
```suggestion
			$this->get_css_assets_url( 'div-block', 'assets/css/', null, true, true ),
```
I see this in other widgets. Can you check if it's needed here?

Reply:
Not needed for this case.